process/preprocess/eventsPreProcessorV1.go

```diff
@@ -88,9 +88,6 @@ func checkBlockDataValid(block *outport.OutportBlock) error {
 	if block.BlockData == nil {
 		return ErrNilBlockData
 	}
-	if block.TransactionPool == nil {
```

Comment:
this nil check is not needed at this point, it will be checked later
- for headers v3 this does  not match anymore